ui/components/multichain/pages/send/components/quote-card/index.tsx

```diff
@@ -90,7 +90,7 @@ export function QuoteCard({ scrollRef }: QuoteCardProps) {
           event: MetaMetricsEventName.sendSwapQuoteReceived,
           category: MetaMetricsEventCategory.Send,
           properties: {
-            is_first_fetch: isQuoteJustLoaded,
+            is_first_fetch: isQuoteJustLoaded ?? false,
```

Comment:
Nit: Since `isQuoteJustLoaded` sounds like it ought to be a boolean, instead of adding `false` here, does it make sense to change this line above:

``` typescript
const isQuoteJustLoaded = bestQuote && prevBestQuote.current === undefined;
```

to this?

``` typescript
const isQuoteJustLoaded = (bestQuote && prevBestQuote.current === undefined) || false;
```